integration/spark2/src/test/scala/org/apache/carbondata/spark/testsuite/booleantype/BooleanDataTypesInsertTest.scala

```diff
@@ -302,79 +302,76 @@ class BooleanDataTypesInsertTest extends QueryTest with BeforeAndAfterEach with
 
     checkAnswer(
       sql("select booleanField,intField from boolean_table2"),
-      Seq(Row(null, 10), Row(null, 17), Row(null, 11),
-        Row(null, 10), Row(null, 10), Row(null, 14),
-        Row(null, 10), Row(null, 10), Row(null, 16), Row(null, 10))
+      Seq(Row(true, 10), Row(true, 17), Row(true, 11),
+        Row(true, 10), Row(true, 10), Row(true, 14),
+        Row(true, 10), Row(true, 10), Row(true, 16), Row(true, 10))
     )
 
     checkAnswer(
       sql("select booleanField,intField,booleanField2 from boolean_table2"),
-      Seq(Row(null, 10, true), Row(null, 17, true), Row(null, 11, true),
-        Row(null, 10, true), Row(null, 10, true), Row(null, 14, false),
-        Row(null, 10, false), Row(null, 10, false), Row(null, 16, false), Row(null, 10, false))
+      Seq(Row(true, 10, true), Row(true, 17, true), Row(true, 11, true),
+        Row(true, 10, true), Row(true, 10, true), Row(true, 14, false),
+        Row(true, 10, false), Row(true, 10, false), Row(true, 16, false), Row(true, 10, false))
     )
   }
 
   test("Inserting with the number of data type in source and target table columns being different, source more than target") {
-    sql(
-      s"""
-         | CREATE TABLE boolean_table(
-         | shortField SHORT,
-         | booleanField BOOLEAN,
-         | intField INT,
-         | bigintField LONG,
-         | doubleField DOUBLE,
-         | stringField STRING,
-         | timestampField TIMESTAMP,
-         | decimalField DECIMAL(18,2),
-         | dateField DATE,
-         | charField CHAR(5),
-         | floatField FLOAT,
-         | complexData ARRAY<STRING>,
-         | booleanField2 BOOLEAN
-         | )
-         | STORED BY 'carbondata'
-         | TBLPROPERTIES('sort_columns'='','DICTIONARY_INCLUDE'='dateField, charField')
-       """.stripMargin)
+    val exception_insert: Exception =intercept[Exception] {
```

Comment:
@shivamasn : why whole test case showing in diff, only modify required content.

Reply:
@ajantha-bhat This test case is not required now..so ignored the test case.